django_tutorial/first_project/firstapp/models.py

```diff
@@ -0,0 +1,330 @@
+# This is an auto-generated Django model module.
+# You'll have to do the following manually to clean this up:
+#   * Rearrange models' order
+#   * Make sure each model has one field with primary_key=True
+#   * Make sure each ForeignKey has `on_delete` set to the desired behavior.
+#   * Remove `managed = False` lines if you wish to allow Django to create, modify, and delete the table
+# Feel free to rename the models, but don't rename db_table values or field names.
+from __future__ import unicode_literals
+
+from django.db import models
+
+
+class CensusZipBusinessPatterns(models.Model):
+    st = models.TextField(blank=True, null=True)
+    zipcode = models.TextField(blank=True, null=True)
+    geo_id = models.TextField(blank=True, null=True)
+    geo_ttl = models.TextField(blank=True, null=True)
+    footid_geo = models.FloatField(blank=True, null=True)
+    naics2012 = models.TextField(blank=True, null=True)
+    naics2012_ttl = models.TextField(blank=True, null=True)
+    footid_naics = models.FloatField(blank=True, null=True)
+    year = models.TextField(blank=True, null=True)
+    empszes = models.TextField(blank=True, null=True)
+    empszes_ttl = models.TextField(blank=True, null=True)
+    estab = models.TextField(blank=True, null=True)
+    estab_f = models.FloatField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'census__zip_business_patterns'
```

Comment:
So this is something that super confused me. How does Django know what the schema is? Or does it just ignore the schema?